### PR TITLE
[UITools] Added shortcut "Start new line below current line" (Shift + Enter)

### PR DESCRIPTION
**Pull request by SlavaRa**
shift_plus_enter_start_new_line_below_current_line

### CODE REVIEW

**Comment by Meychi** [comment]
I'm not sure about this. We have similar hard coded functionality for this already somewhere. If i remember correctly, for example inserting br in html.

**Comment by SlavaRa** [comment]
No, I did it in May, but PR was not merge for half a year, I closed it

**Comment by elsassph** [comment]
There is `Shift+Enter` inside a string splits it by inserting closing quote, `+`, opening quote on next line. But it also seem nice to have - maybe another shortcut?

File: FlashDevelop/Dialogs/ShortcutDialog.cs [hunk 1]
```diff
@@ -614,7 +614,7 @@ private void SaveCustomShortcut(object sender, EventArgs e)
         /// </summary>
         private void CloseButtonClick(object sender, EventArgs e)
         {
-            this.Close();
+            //this.Close();
```
Comment (elsassph) [inline FlashDevelop/Dialogs/ShortcutDialog.cs]: Really?

Comment (SlavaRa) [reply FlashDevelop/Dialogs/ShortcutDialog.cs]: Oh.. moment... Thanks

**Comment by SlavaRa** [comment]
> There is Shift+Enter inside a string splits it by inserting closing quote, +, opening quote on next line.
In FD ?

**Comment by elsassph** [comment]
Yes, try it :)

**Comment by SlavaRa** [comment]
Not works

**Comment by elsassph** [comment]
Doesn't work in JS file but doe in HX

**Comment by elsassph** [comment]
I have a dev build from August.

**Comment by SlavaRa** [comment]
Only single quotes ;)

**Comment by elsassph** [comment]
Oh only single quote is a shame - didn't notice it because I'm using only single quotes :D 
So how do we progress that? I kind of like the strings splitting feature.

**Pull request closed without merging**